cmd/future/bump_deps.go

```diff
@@ -27,21 +45,54 @@ var BumpDeps = &cobra.Command{
 		updateSchema(deps, &s)
 
 		if err := composer.WriteComposerJson(file, s); err != nil {
-			log.Fatalf("could not write composer.json: %v\n", err)
-			return
+			msg := fmt.Sprintf("could not write composer.json: %v\n", err)
+
+			_, err := client.Push(cmd.Context(), &collector.PushRequest{
+				Command: cmd.Name(),
+				Output:  msg,
+				Status:  1,
+			})
+
+			if err != nil {
+				log.Fatal(msg)
+			}
+
+			os.Exit(1)
 		}
 
 		if len(deps.Installed) == 0 {
-			log.Print("all dependencies are at their latest version - nothing to update\n")
-			return
+			_, err := client.Push(cmd.Context(), &collector.PushRequest{
+				Command: cmd.Name(),
+				Output:  "all dependencies are at their latest version - nothing to update\n",
+				Status:  0,
+			})
+
+			if err != nil {
+				log.Printf("%+v", err)
+			}
+
+			os.Exit(0)
 		}
 
-		log.Print("successfully updated the following dependencies in the composer.json file:\n")
+		builder := strings.Builder{}
```

Comment:
`var builder strings.Builder` unless it has a constructor like `builder := strings.NewBuilder()`